services/director-v2/tests/unit/test_models_schemas_constants.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import re
+
+import pytest
+from simcore_service_director_v2.models.schemas.constants import (  
+    # TODO: add test with REGEX_DY_SERVICE_PROXY,
+    # TODO: add test with  REGEX_DY_SERVICE_SIDECAR,
+    DYNAMIC_SIDECAR_DOCKER_IMAGE_RE,
+)
+
+
+@pytest.mark.parametrize(
+    "sample",
+    [
+        "itisfoundation/dynamic-sidecar:staging-github-staging_diolkos1-2022-06-15--15-04.75ddf7e3fb86944ef95fcf77e4075464848121f1",
```

Comment:
and maybe add as parameter the release tag version as well... just to be on the safe side